addon/plugins/table/index.ts

```diff
@@ -29,14 +29,27 @@ export class TableView extends PluginTableView {
     public cellMinWidth: number,
   ) {
     super(node, cellMinWidth);
-    this.addClasses(node);
+    this.addAttrs(node);
   }
 
-  private addClasses(node: Node): void {
-    const nodeClasses = node.attrs.class as string | undefined;
+  private addAttrs(node: Node): void {
+    const {
+      class: nodeClasses,
+      style,
+      ...attrs
+    } = node.attrs as Record<string, unknown>;
     if (typeof nodeClasses === 'string') {
       this.table.classList.add(...nodeClasses.split(' '));
     }
+    if (typeof style === 'string') {
+      this.table.style.cssText = `${this.table.style.cssText} ${style}`;
+    }
+    for (const attrName in attrs) {
+      const attr = attrs[attrName];
+      if (typeof attr === 'string') {
+        this.table.setAttribute(attrName, attr);
+      }
+    }
```

Comment:
I don't know if we necessarily want this, as table `attrs` will not always be valid html attributes.